app/javascript/components/concept-exercise-graph/Concept.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { CompleteIcon } from './CompleteIcon'
+
+import { IConcept, ConceptState } from './concept-types'
+
+export const Concept = ({
+  index,
+  slug,
+  conceptExercise,
+  uuidOfExercise,
+  status,
+  handleEnter,
+  handleLeave,
+  isActive,
+  isDimmed,
+  adjacentConcepts,
+}: IConcept & {
+  isActive: boolean
+  isDimmed: boolean
+  adjacentConcepts: string[]
+}) => {
+  const name = slugToTitlecase(slug)
+
+  let classes = 'card'
+  classes += ` card-${status}`
+  classes += isActive ? ' card-active' : ''
+  classes += isDimmed ? ' dim' : ''
+  adjacentConcepts.forEach((adjacentConcept: string): void => {
+    classes += ` adjacent-to-${adjacentConcept}`
+  })
```

Comment:
You might want to build an array here, and then join the array on `" "`, rather that having the spaces in each string as you build it.

Reply:
That is the future proofed way, when this section was shorter just 2 lines, this made more sense. 

Can update it.